hacksquad_bot/utils.py

```
@@ -18,10 +17,32 @@ async def get_team(slug):
     async with aiohttp.ClientSession() as session:
         async with session.get(f'https://www.hacksquad.dev/api/team/?id={slug}') as response:
             if response.status == 200:
-                response = await response.text()
-                response = json.loads(response)
+                response = await response.json()
                 team = response.get("team")
                 return team
 
             else:
-                return None
\ No newline at end of file
+                return None
+
+async def get_contributors():
+    async with aiohttp.ClientSession() as session:
+        async with session.get('https://contributors.novu.co/contributors') as response:
+            if response.status == 200:
+                response = await response.json()
+                contributors = response.get("list")
+                return contributors
+
+            else:
+                return None
+
+async def get_contributors_mini():
+    # I do not think that we would get much of a performance benefit from this but leaving it here all the same
+    async with aiohttp.ClientSession() as session:
+        async with session.get('https://contributors.novu.co/contributors-mini') as response:
+            if response.status == 200:
+                response = await response.json()
+                contributors = response.get("list")
+                return contributors
+
+            else:
+                return None
```

You should raise a custom exception instead